### PR TITLE
Fluff item for Captain Cardgage

### PR DESCRIPTION
**Pull request by FlattestGuitar**
No CL, because I'm lazy and backstage.

Conversion kit, turns a welding helmet into its flamed version.

Unfortunately, because of how the icon change works, it's impossible to change it sensibly.

This deletes the current one and spawns a new one. Also transfers prints.

### CODE REVIEW

**Comment by taukausanake** [comment]
Hmmm, isn't the flame welding mask standard in /tg/? I guess there's no reason we would need them ourselves but just pointing that out. Or maybe they were copied over and I didn't notice?

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -225,6 +225,29 @@
...
+	desc = "Some spraypaint and a stencil, perfect for painting flames onto a welding helmet!"
+	icon_state = "modkit"
+	w_class = 2
+	force = 0
+	throwforce = 0
+
+/obj/item/device/fluff/cardgage_helmet_kit/afterattack(atom/target, mob/user, proximity)
+	if(!proximity || !ishuman(user) || user.lying)
+		return
+	var/mob/living/carbon/human/H = user
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: Why are you typecasting this as human?

**Comment by FlattestGuitar** [comment]
God, I don't even know what happened there. Two different computers don't play well with GitHub for me.

The cast was left over from when I was calling `update_icons()` on the user, just in case they were wearing the thingy, but I changed it for the reasons mentioned earlier.

**Comment by Fox-McCloud** [comment]
@FreeStylaLT @TheDZD 

Will need to be added to the table.

**Pull request merged**